src/amo/store.js

```diff
@@ -7,10 +7,13 @@ import api from 'core/reducers/api';
 import auth from 'core/reducers/authentication';
 import reviews from 'amo/reducers/reviews';
 import search from 'core/reducers/search';
+import version from 'core/reducers/version';
 
 export default function createStore(initialState = {}) {
   return _createStore(
-    combineReducers({ addons, api, auth, search, reviews, reduxAsyncConnect }),
+    combineReducers({
+      addons, api, auth, search, reviews, reduxAsyncConnect, version,
```

Comment:
I think it's confusing to call this `version`. It would make more sense to me as `versions` since it's a map of version objects by ID.